css/templates/lune/page-vars.css

```diff
@@ -62,17 +63,22 @@
   --color-font-primary: var(--color-dark-grey-1);
   --color-font-secondary: var(--color-font-primary);
 
+  /* Animations  */
+  --animation-delay-start: .4s;
+  --animation-delay-step: .1s;
+
   /*
    * Copying these @custom-media rules and the --space-* variables from
-   * bass-override-vars.css. This is a temporary solution, until we merge
-   * all the CSS into one file.
+   * bass-override-vars.css. This is a temporary solution, waiting for a
+   * better way of doing it.
    */
   @custom-media --breakpoint-xs (max-width: 40rem);
```

Comment:
not sure why you have to do this , it should work by default without needing to add this here again.